src/xrGame/Actor.cpp

```diff
@@ -395,6 +395,7 @@ void CActor::Load(LPCSTR section)
                 Msg("~ [%s] is missing sounds for type [%s]", hit_snd_sect, hit_name);
             }
 #endif
+            sndHit->reserve(cnt);
```

Comment:
This is wrong, read the code carefully, please

Reply:
Fix it https://github.com/OpenXRay/xray-16/pull/1409/commits/1fcc195aeef0396e6977db8ef7543cdc1c594137